### PR TITLE
Add unit tests for grafana_plugin

### PR DESCRIPTION
**Pull request by rrey**
Adding very minimal tests that covers issues reported on `grafana_plugin` during plugin version parsing.

### CODE REVIEW

File: plugins/modules/grafana_dashboard.py [hunk 1]
```diff
@@ -67,10 +67,11 @@
       - Revision of the public grafana dashboard to import
     default: 1
     version_added: '2.10'
-  message:
+  msg:
```
Comment (seuf) [inline plugins/modules/grafana_dashboard.py]: This change should be in an other PR I think..

File: plugins/modules/grafana_plugin.py [hunk 2]
```diff
@@ -78,6 +78,11 @@ class GrafanaCliException(Exception):
     pass
 
 
+def parse_version(string):
```
Comment (seuf) [inline plugins/modules/grafana_plugin.py]: Nice :)

File: tests/unit/modules/grafana/grafana_plugin/test_grafana_plugin.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class GrafanaPlugin(TestCase):
+
+    @patch('ansible_collections.community.grafana.plugins.modules.grafana_plugin.grafana_cli_bin')
+    def test_plugin_install_zip(self, mock_grafana_cli_bin):
+        mock_grafana_cli_bin.return_value = "grafana-cli plugins"
+
+        params = {
+            "name": "alexanderzobnin-zabbix-app"
```
Comment (seuf) [inline tests/unit/modules/grafana/grafana_plugin/test_grafana_plugin.py]: version should be specified too ?

File: plugins/modules/grafana_dashboard.py (hunk 1 above)
Comment (rrey) [reply plugins/modules/grafana_dashboard.py]: yeah probably, I faced this issue while pushing the tests.
In fact we have to port the changes made in this patch https://github.com/ansible/ansible/pull/60051/files

File: tests/unit/modules/grafana/grafana_plugin/test_grafana_plugin.py (hunk 3 above)
Comment (rrey) [reply tests/unit/modules/grafana/grafana_plugin/test_grafana_plugin.py]: The version is not provided through the module when installing from zip right ? or is it ?

**Comment by gundalow** [comment]
Doesn't look like it rebased

**Comment by gundalow** [comment]
Ah rebase failed due to merge conflict
```
Auto-merging plugins/callback/grafana_annotations.py
CONFLICT (add/add): Merge conflict in galaxy.yml
Auto-merging galaxy.yml
error: Failed to merge in the changes.
hint: Use 'git am --show-current-patch' to see the failed patch
Patch failed at 0001 Initial Commit
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
```

**Pull request merged**